infra/charts/management-service/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "0.0.1"
+appVersion: v0.6.0
```

Comment:
Besides this, we should keep `xpManagement.image.tag` updated for every release. The `appVersion` here is less relevant. The same applies to the treatment service.